src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+
+  const sourceObject = sourceArray.reduce((acc, curr) => {
+    const entry = curr.split(':').map((item) => item.trim());
```

Comment:
The 'map' function is being used to trim items, which is fine, but ensure that there are always two items (property and value) before destructuring to avoid potential errors with incomplete style declarations.